wpilibc/src/test/native/cpp/simulation/DCMotorSimTest.cpp

```diff
@@ -34,8 +34,7 @@ TEST(DCMotorSimTest, VoltageSteadyState) {
     // Then, SimulationPeriodic runs
     frc::sim::RoboRioSim::SetVInVoltage(
         frc::sim::BatterySim::Calculate({sim.GetCurrentDraw()}));
-    sim.SetInputVoltage(motor.Get() *
-                        frc::RobotController::GetBatteryVoltage());
+    sim.SetInputVoltage(motor.Get() * 6_V);
```

Comment:
This was fine as is. The issue is the Set(12_V) call.

Reply:
Reverting this back and changing motor.SetVoltage(12_V) to 6_V